doc/source/param/initial.incl

```diff
@@ -343,6 +352,41 @@ the columns of the matrix given in equation B3 of`
   * ``"fast"`` :e:`A linear fast magnetosonic wave.`
   * ``"slow"`` :e:`A linear slow magnetosonic wave.`
 
+Jeans Wave
+~~~~~~~~~~
+
+:e:`To initialize a Jeans wave, set this parameter to` ``"jeans"``.
+:e:`We use equations consistent with what Athena and (earlier
+versions of) Athena++ use. In detail, we use:`
+
+.. math::
+   \rho     &= \rho_{\rm bkg}
+          \left( 1 + A \sin\left(\frac{2\pi}{\lambda} x_0 \right) \right) \\
+   \rho v_0 &= \rho_{\rm bkg} \frac{\sqrt{|\omega^2|}}{2\pi / \lambda} A
+          \begin{cases}
+                0                       & \omega^2 > 0 \\
+                \cos\left(\frac{2\pi}{\lambda} x_0 \right) & \omega^2 < 0 \\
+          \end{cases} \\
+   v_1      &= 0 \\
+   v_2      &= 0 \\
+   \rho E   &= \frac{P_{\rm bkg}}{\gamma - 1} + \gamma A \sin\left(\frac{2\pi}{\lambda} x_0\right) \\
```

Comment:
Actually, in looking at the code, amplitude is really A * background, but that seems OK because the background values are always set to 1.

Reply:
I'm not sure I follow. We initialize the object representing the $\rho E$ equation on line 869 of `enzo_EnzoInitialInclinedWave.cpp`. For this particular case, this object computes:
`background + ampiltude * eigenvalue * std::sin(2*pi*x0/lambda` where: 
- `background` is $\frac{P_{\rm bkg}}{\gamma - 1}$
- `amplitude` is unit-less (usually it's 1e-6)
- `eigenvalue` is equal to $\gamma$.

The code could definitely be a lot cleaner and simpler. I started to work on this in this PR but I didn't get very far.

The equations to initialize the primitives are a lot simpler. If you prefer, we could switch to those.

Reply:
This is just a minor presentation issue in the docs, so totally unimportant, but just to try and clear up what I meant. Your value `amplitude` in the code snippet you gave really has the same units as `p_bkg` because it is added directly to background (rather than `background * (1 + amplitude * eigenvalue * send::sun(2*pi*x0/lambda))`). That works because the standard background is unitless and `p_bkg/rho_bkg = gamma`. I suppose, one might worry that if the background value where changed, then we might get into problems, but in the code, the background is hardcoded at 1.0, which is perfectly sensible for a test. So, to be clear, my suggested change is wrong because it doesn't match with the code (I wrote it before looking at the code).

Reply:
Gotcha, I see.